frontend/src/Components/CreateCommentComponent.tsx

```diff
@@ -205,16 +206,22 @@ export default function CreateCommentComponent(props: CreateCommentProps) {
     const handleAnswer = () => {
         setPosting(true);
         props.onAnswer(answerText, props.post, props.comment)
-            .then(() => {
+            .then((response: string | CommentInfo | undefined) => {
                 setStorageValueDebounced('');
                 setStorageValueDebounced.flush();
-                setAnswerText('');
+                if (!props.staticEditor) {
+                    setAnswerText('');
+                } else {
+                    if (response && typeof response === 'string') {
+                        setPreviewing(response);
+                    }
+                }
             })
             .catch(error => {
                 console.log('onAnswer ERR', error);
             })
             .finally(() => {
-                setPreviewing(null);
+                //setPreviewing(null);
```

Comment:
Why is this commented out?

Reply:
debug leftovers, thanks for catching this